libdirections-onboard/api/current.txt

```diff
@@ -0,0 +1,17 @@
+// Signature format: 3.0
+package com.mapbox.navigation.route.onboard {
+
+  @com.mapbox.annotation.module.MapboxModule(type=MapboxModuleType.NavigationOnboardRouter) public final class MapboxOnboardRouter implements com.mapbox.navigation.base.route.Router {
+    ctor public MapboxOnboardRouter(String accessToken, com.mapbox.navigation.navigator.internal.MapboxNativeNavigator navigatorNative, com.mapbox.navigation.base.options.OnboardRouterOptions options, com.mapbox.base.common.logger.Logger logger, com.mapbox.navigation.base.internal.accounts.SkuTokenProvider skuTokenProvider);
+    method public void cancel();
+    method public void getRoute(com.mapbox.api.directions.v5.models.RouteOptions routeOptions, com.mapbox.navigation.base.route.Router.Callback callback);
+    method public void getRouteRefresh(com.mapbox.api.directions.v5.models.DirectionsRoute route, int legIndex, com.mapbox.navigation.base.route.RouteRefreshCallback callback);
+    method public void shutdown();
+    field public static final com.mapbox.navigation.route.onboard.MapboxOnboardRouter.Companion! Companion;
+  }
+
+  public static final class MapboxOnboardRouter.Companion {
```

Comment:
Move `private const`s out from the `companion object` into the file to save 1 unnecessary class creation (micro-optimization) as outer class (file) is always created